onnxruntime/python/tools/transformers/convert_generation.py

```diff
@@ -2165,19 +2199,29 @@ def test_gpt_model(args: argparse.Namespace, sentences: Optional[List[str]] = No
         prefix_vocab_mask = np.ones((batch_size, vocab_size), dtype=np.int32)
         inputs["prefix_vocab_mask"] = prefix_vocab_mask
 
-    logger.debug("ORT inputs", inputs)  # noqa: PLE1205
-    result = ort_session.run(None, inputs)
-
     if args.save_test_data:
         test_data_dir = Path(args.output).parent.as_posix()
         logger.debug("test_data_dir", test_data_dir)  # noqa: PLE1205
         from bert_test_data import output_test_data
 
+        logger.info(f"Saving test_data to {test_data_dir}/test_data_set_* ...")
+
         all_inputs = [inputs]
         for i, inputs in enumerate(all_inputs):
             dir = os.path.join(test_data_dir, "test_data_set_" + str(i))
             output_test_data(dir, inputs)
 
+    logger.debug("ORT inputs", inputs) # noqa: PLE1205
```

Comment:
```suggestion
    logger.debug("ORT inputs", inputs)
```

fix the lint instead?